### ISSUE TITLE
Utils.numToBytes(long value, boolean withCRLF) always return null if withCRLF is a false value

### ISSUE THREAD

**Issue by tasszz2k** [netease-im/camellia#46]
why does this method always return null if `withCRKF` input is a `false` value
```        
        long originalNum = 10;
        byte[] bytes = Utils.numToBytes(originalNum, false); // --> bytes = null ?
````

**Comment by caojiajun** [comment]
it's a bug, but in proxy logic, withCRKF always setting true.

**Comment by caojiajun** [comment]
https://github.com/netease-im/camellia/commit/a7c746d05d3994238200eff9108cca9d57247c48

this bug will be fixed in 1.1.3

**Comment by tasszz2k** [comment]
thank you for your support 👍 
:"D
